__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -308,3 +564,18 @@ export const Scenarios = (args: PropsFor<typeof LabeledField>) => {
         </View>
     );
 };
+
+/**
+ * Here is an example where LabeledField is used with a custom element.
```

Comment:
**suggestion:** What do you think about adding some NOTE here for folks to be aware that this is available but that we still suggest using WB fields as much as they can?